test/config.test.ts

```diff
@@ -51,6 +51,7 @@ function assertSchema(
         const validator = new Ajv({
             // skip full check if we are not encoding refs
             validateFormats: config.encodeRefs === false ? undefined : true,
+            allowUnionTypes: true,
```

Comment:
If this ch age is not necessary, please remove it.

Reply:
done